packages/jasmine/BUILD.bazel

```diff
@@ -29,6 +29,13 @@ genrule(
     visibility = ["//docs:__pkg__"],
 )
 
+genrule(
+    name = "npm_version_check",
+    srcs = ["//:tools/npm_version_check.js"],
+    outs = [":npm_version_check.js"],
+    cmd = "cp $< $@",
```

Comment:
we can drop the bash requirement here using the copy_file helper from skylib - I'll add that as separate commit